client/lib/create-selector/test/index.js

```diff
@@ -1,22 +1,19 @@
 /**
  * External dependencies
  */
-import Chai, { expect } from 'chai';
+import { expect } from 'chai';
 import filter from 'lodash/filter';
 import sinon from 'sinon';
-import sinonChai from 'sinon-chai';
 
 /**
  * Internal dependencies
  */
 import createSelector from '../';
 
-describe( '#createSelector', () => {
+describe( 'index', () => {
 	let selector, getSitePosts;
 
 	before( () => {
-		Chai.use( sinonChai );
```

Comment:
It applies it globally anyway, so better to have it in one place.

Yes, it uses sinon-chai like here:
```js
expect( getDeps ).to.have.been.calledWithExactly( state, 1, 2, 3 );
```